### PR TITLE
rename some files in scripts/

### PR DESCRIPTION
**Pull request by ronuchit**

### CODE REVIEW

**Comment by tomsilver** [comment]
I think so? but I don't care that much

File: scripts/analyze_results_directory.py [hunk 1]
```diff
@@ -157,8 +157,8 @@ def _main() -> None:
     pd.set_option("expand_frame_repr", False)
     print("\n\nAGGREGATED DATA OVER SEEDS:")
     print(means)
-    means.to_csv("supercloud_analysis.csv")
-    print("\n\nWrote out table to supercloud_analysis.csv")
+    means.to_csv("results_analysis.csv")
```
Comment (tomsilver) [inline scripts/analyze_results_directory.py:160]: maybe `results_summary.csv`? we're not really analyzing anything

File: scripts/supercloud/run_active_learning_baseline.sh [hunk 2]
```diff
@@ -3,7 +3,7 @@
 START_SEED=456
 NUM_SEEDS=10
 CYCLES=100
-FILE="scripts/submit.py"
+FILE="scripts/supercloud/submit_supercloud_job.py"
```
Comment (tomsilver) [inline scripts/supercloud/run_active_learning_baseline.sh:6]: jobs?

**Review by tomsilver — APPROVED** [review]

File: scripts/supercloud/run_active_learning_baseline.sh (hunk 2 above)
Comment (ronuchit) [reply scripts/supercloud/run_active_learning_baseline.sh:6]: no, it just does one job

**Pull request merged**